src/code/models/simulation-control.ts

```diff
@@ -9,9 +16,9 @@ export const  SimulationControl = types.model(
     isPlaying: types.optional(types.boolean, false),
     time: types.maybe(types.Date),
     halfTime: types.maybe(types.Date),
-    timeStep: types.optional(types.number, 15),      // minutes per time step
-    timeScale: types.optional(types.number, 60),     // wall-time to simulation time mulitplier.
-    updateIntervalS: types.optional(types.number, 5), // updates frame rate.
+    timeStep: types.optional(types.number, 5),      // minutes per time step
+    timeScale: types.optional(types.number, 300),     // wall-time to simulation time multiplier.
```

Comment:
Since you have created these nice configuration constants, maybe it would be nice to use them here too. eg  `types.optional(types.number, kOverrides. timeScale)`

Reply:
I like it -- keep it DRY. I'll make the change.